skaffold/src/main/resources/messages/SkaffoldBundle.properties

```diff
@@ -42,3 +42,4 @@ skaffold.override.image.repo.tooltip=<html>The default image repository allows y
 skaffold.no.file.selected.error=Skaffold configuration file is not selected.
 skaffold.invalid.file.error=Skaffold configuration file doesn't exist or invalid.
 skaffold.corrupted.run.settings=Your Skaffold run configuration is corrupted. Please re-create it to fix.
+skaffold.not.on.system.error=Could not detect Skaffold installation. Please install Skaffold. See required dependencies <a href="https://github.com/GoogleContainerTools/google-container-tools-intellij#prerequisites-and-required-dependencies">here</a>:
```

Comment:
Does this link work in the message? I see it ends with a colon, is this intentional?

Reply:
It probably should not end with a colon. Also the link does not work. Probably need to attach a URL handler.

Reply:
I'd suggest, if its not easy to figure out how to get the URL to work here, to just change it from a hyperlink to just print the url (so the user can copy/paste it), and open a separate issue to track getting the link working.

Reply:
Agree, might be not easy here. We can also think of some sort of notification - they are more flexible.